app/jobs/fs/download_sent_message_related_messages_job.rb

```diff
@@ -1,18 +1,17 @@
 module Fs
   class DownloadSentMessageRelatedMessagesJob < ApplicationJob
-    include GoodJob::ActiveJobExtensions::Concurrency
-
-    good_job_control_concurrency_with(
-      perform_throttle: [250, 1.minute]
-    )
-
     class MissingRelatedMessagesError < StandardError
     end
 
+    class TemporaryMissingRelatedMessagesError < StandardError
+    end
+
     retry_on MissingRelatedMessagesError, attempts: 1 do |_job, _error|
       # no-op
     end
 
+    retry_on TemporaryMissingRelatedMessagesError, wait: ->( executions ) { executions * 5.minutes }
```

Comment:
Pridala som sem este retries. "Problem" aktualne je nasledovny:

- po stiahnuti odoslaneho podania sa naplanuje stahovanie potvrdenky (suvisiacich sprav) o 3 minuty
- ak ale potvrdenka este neexistuje (stava sa bezne, ze to trva dlhsie ako 3 minuty), tak sa dotiahne az pri najblizsom syncu - dlho trva, klienti obcas pisu/volaju, ze co sa deje
- takze som pridala nejake retries, ak do 3 minut tam nie je.

Logika je nasledovna (ak potvrdenka nepride do hodiny):
- prvy pokus 3 min po podani
- dalsi pokus o 5 min
- dalsi o 10 min
- dalsi o 15 min
- dalsi o 20 min
- fail